build/KoreBuild.ps1

```diff
@@ -122,12 +122,14 @@ $msbuildArtifactsDir = "$repoFolder/artifacts/msbuild"
 $msbuildLogFilePath = "$msbuildArtifactsDir/msbuild.binlog"
 $msBuildResponseFile = "$msbuildArtifactsDir/msbuild.rsp"
 
+# Workaround https://github.com/Microsoft/msbuild/issues/2168
+$sdkPath = ((dotnet --info | sls 'Base Path') -split ':' | select-object -last 1).Trim()
```

Comment:
We also have `DotnetInstallationFolder\sdk\DotnetVersion\` to figure out this path. 😉 
Either way is fine though.